firestore/collection/index.ts

```diff
@@ -185,16 +185,26 @@ const filterEmptyUnlessFirst = <T = unknown>(): UnaryFunction<
     map(([_, current]) => current)
   );
 
+type SnapshotListenOptions = firebase.firestore.SnapshotListenOptions;
+type Options = {
+  events?: DocumentChangeType[],
+  snapshotListenOptions?: SnapshotListenOptions,
+};
+
+const DEFAULT_SNAPSHOT_LISTEN_OPTIONS = { includeMetadataChanges: true};
+
 /**
  * Return a stream of document changes on a query. These results are not in sort order but in
  * order of occurence.
  * @param query
  */
 export function collectionChanges<T=DocumentData>(
   query: Query<T>,
-  events: DocumentChangeType[] = ALL_EVENTS
+  options: Options = {}
 ): Observable<DocumentChange<T>[]> {
-  return fromCollectionRef(query, { includeMetadataChanges: true }).pipe(
+  const events = options.events || ALL_EVENTS;
+  const snapshotListenOptions = options.snapshotListenOptions ?? DEFAULT_SNAPSHOT_LISTEN_OPTIONS;
```

Comment:
may want to spread these instead, so that if someone specifies some options but not others, it doesn't get confused:

```ts
const snapshotListenOptions = {
  ...DEFAULT_SNAPSHOT_LISTEN_OPTIONS,
  ...options.snapshotListenOptions,
};
```